Applications/VpView/vpTrackIO.cxx

```diff
@@ -134,3 +134,21 @@ void vpTrackIO::AddTrack(vtkVgTrack* track)
   this->TrackModel->AddTrack(track);
   track->FastDelete();
 }
+
+//-----------------------------------------------------------------------------
+int vpTrackIO::GetTrackTypeIndex(const char* typeName)
+{
+  const auto index = this->TrackTypes->GetTypeIndex(typeName);
+
+  if (index >= 0)
+    {
+    return index;
+    }
+
+  vgTrackType type;
+  type.SetId(typeName);
+  type.SetColor(0.0, 0.0, 0.0);
```

Comment:
Black?  Maybe use default that comes up in the dialog for new type

Reply:
Okay, removed this line. (The dialog just uses what the `vgTrackType` ctor sets...)